packages/design-system/src/components/dropdown-menu.tsx

```diff
@@ -9,7 +9,8 @@ import { panelStyles } from "./panel";
 
 export const DropdownMenu = DropdownMenuPrimitive.Root;
 export const DropdownMenuArrow = styled(DropdownMenuPrimitive.Arrow, {
-  fill: "$panel",
+  fill: "$colors$slate4",
+  stroke: "$colors$slate1",
```

Comment:
We should not use color tokens directly but rather semantic alternatives. What are you trying to express here?